eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/utils/Color.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2017 SchedJoules
+ *
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.schedjoules.eventdiscovery.framework.utils;
+
+import android.content.Context;
+import android.content.res.Resources;
+import android.support.annotation.AttrRes;
+import android.util.TypedValue;
+
+
+/**
+ * Represents the Color referenced by a specific attribute. The only supported method is {@link #intValue()}.
+ * <p>
+ * TODO: with Android O you can create meaningful instances of {@link android.graphics.Color}. We probably can use this as the superclass then.
+ *
+ * @author Marten Gajda
+ */
+public final class Color extends Number
```

Comment:
I like this idea to model color, but it's not really a number for me. Could we maybe have a dedicated interface?
With a single method currently, something like `int rgb()` - not sure what representation is that int is actually, but a method name that refers to that representation.
And then we could also implement `ContextCompat.getColor(context, colorRes)` underneath. So maybe 2 implementations, `ResourceColor` and `ThemeAttributeColor`.